File: 1. two-sum/two-sum.py
def twoSum(nums: list[int], target: int) -> list[int]:
    for i in range(0, len(nums)):
        for  j in range(0, len(nums)):
            if i == j:
                continue
            else:
                mySum = nums[i] + nums[j]
                if mySum == target:
                    return [i, j]
    return []


print(twoSum([6, 4, 7, 3], 9))

File: 1. two-sum/two-sum.js
function twoSum (nums, target) {
    for (let i = 0; i < nums.length; i++) {
        for (let j = 0; j < nums.length; j++) {
            if (i == j) {
                continue;
            } else {
                let mySum = nums[i] + nums[j];
                if (mySum == target) {
                    return [i, j];
                }
            }
        }
    }
    return [];
};

console.log(twoSum([3, 6, 7, 1], 9));